machine/v1/types_nutanixprovider.go

```diff
@@ -86,6 +89,12 @@ type NutanixMachineProviderConfig struct {
 	// credentials data to access Nutanix PC client
 	// +kubebuilder:validation:Required
 	CredentialsSecret *corev1.LocalObjectReference `json:"credentialsSecret"`
+
+	// failureDomainName is the name of the FailureDomain with which this Machine is associated.
+	// If this is configured, the Nutanix machine controller will use the prism-central endpoint
+	// and credentials defined in the named FailureDomain to communicate to the prism_central.
```

Comment:
is prism central to be referred to as prism-central or prism_central?

Reply:
Fixed, now refer to prism_central consistently.